### ISSUE TITLE
Error on https://game192194.konggames.com/games/player_03/run-3/frame/12db9dad-a949-42b2-b7d2-065cbb15c7d0/

### ISSUE THREAD

**Issue by Nameless-GitHubber** [ruffle-rs/ruffle#11092]
# Error Info
Error name: Error
Error message: panicked at 'Vertex shader failed to compile: Unimplemented("Unsupported conversion from Float1 to float4")', render/wgpu/src/context3d/current_pipeline.rs:437:10
Error stack:
```
Error: panicked at 'Vertex shader failed to compile: Unimplemented("Unsupported conversion from Float1 to float4")', render/wgpu/src/context3d/current_pipeline.rs:437:10
    at W.n.wbg.__wbg_new_6d6ba2e6fc178ce1 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.b7be07493ebf266efa96.js:1:50371)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h2f89ea248b2b086f (wasm://wasm/02c3216e:wasm-function[8575]:0x830bd2)
    at std::panicking::rust_panic_with_hook::h7f7102b82d51338f (wasm://wasm/02c3216e:wasm-function[5600]:0x754b05)
    at core::panicking::panic_fmt::he4489d678d6570d5 (wasm://wasm/02c3216e:wasm-function[8426]:0x82bf6c)
    at core::result::unwrap_failed::h9c69c0d7a98df92b (wasm://wasm/02c3216e:wasm-function[6499]:0x7991c0)
    at <ruffle_render_wgpu::context3d::WgpuContext3D as ruffle_render::backend::Context3D>::process_command::h5ec0ce0ba75a0e26 (wasm://wasm/02c3216e:wasm-function[529]:0xca7ed)
    at ruffle_core::avm2::globals::flash::display3D::context_3d::draw_triangles::hccfb44cd577e1ef9 (wasm://wasm/02c3216e:wasm-function[2366]:0x5527e5)
    at ruffle_core::avm2::function::Executable::exec::h44efd0f4479c6442 (wasm://wasm/02c3216e:wasm-function[797]:0x2b2f7b)
    at ruffle_core::avm2::object::TObject::call_property::h8942575856e474b1 (wasm://wasm/02c3216e:wasm-function[1123]:0x389031)
    at <ruffle_core::avm2::object::Object as ruffle_core::avm2::object::TObject>::call_property::h23afa38f2922fb62 (wasm://wasm/02c3216e:wasm-function[2697]:0x5beef3)
```
AVM2 stack:
```
    at flash.display3D::Context3D/flash::display3D::Context3D::drawTriangles()
    at nme3D.model::ModelGroup/render()
    at nme3D.model::ModelCollection/render()
    at nme3D::Context3DUtils$/render()
```

# Player Info
Allows script access: true
Renderer: wgpu
Adapter Backend: Gl
Adapter Name: "ANGLE (Intel, Intel(R) HD Graphics Direct3D11 vs_5_0 ps_5_0, D3D11)"
Adapter Device Type: IntegratedGpu
Adapter Driver Name: ""
Adapter Driver Info: ""
Enabled features: Features(TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES | PUSH_CONSTANTS)
Available features: Features(TEXTURE_COMPRESSION_BC | CLEAR_TEXTURE | MULTIVIEW)
Current limits: Limits { max_texture_dimension_1d: 16384, max_texture_dimension_2d: 16384, max_texture_dimension_3d: 2048, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 640, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 256, min_storage_buffer_offset_alignment: 256, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: high
Surface samples: 4
Surface size: Extent3d { width: 550, height: 400, depth_or_array_layers: 1 }

# Page Info
Page URL: https://game192194.konggames.com/games/player_03/run-3/frame/12db9dad-a949-42b2-b7d2-065cbb15c7d0/?kongregate_host=www.kongregate.com
SWF URL: https://game192194.konggames.com/gamez/0019/2194/live/Run3.swf?kongregate_game_version=1669305919

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-05-17
Channel: nightly
Built: 2023-05-17T00:20:26.307Z
Commit: 75569e1311807be865ad1d0c60d5f3b151632421
Is extension: false

# Metadata
width: 800
height: 600
frameRate: 60
numFrames: 2
swfVersion: 17
backgroundColor: #000000
isActionScript3: true
uncompressedLength: 14964692

**Comment by n0samu** [comment]
Fixed by #11109

**Comment by Nameless-GitHubber** [comment]
Yay! :D
